### PR TITLE
Added React information

### PR DESCRIPTION
**Pull request by thecodingaviator**
Added React information

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: client/src/pages/guide/english/certifications/front-end-libraries/react/index.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ title: React
 ---
 ## React
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/quadratic-equations/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+![React](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/200px-React-icon.svg.png)
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+React or ReactJS is a frone-end library which is used to create beautiful, reusable and interactive User Interfaces.
```
Comment (ezioda004) [inline client/src/pages/guide/english/certifications/front-end-libraries/react/index.md]: Typo, it should be front-end.

File: client/src/pages/guide/english/certifications/front-end-libraries/react/index.md [hunk 2]
```diff
@@ -3,11 +3,18 @@ title: React
...
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+React is written in JavaScript and JSON( JavaScript Object Notation ). It is designed with reusability in mind and follows a component based pattern, which means write once and use infinitely.
```
Comment (ezioda004) [inline client/src/pages/guide/english/certifications/front-end-libraries/react/index.md]: Did you mean JSX instead of JSON?

**Review by ezioda004 — CHANGES_REQUESTED** [review]
Please make the changes as requested.

**Comment by thecodingaviator** [comment]
@ezioda004 Done

**Review by amandeepmittal — APPROVED** [review]

**Review by ezioda004 — APPROVED** [review]
LGTM

**Comment by ezioda004** [comment]
Thank you @thecodingaviator for your contribution.

**Pull request merged**